Removing GitLab links

tom114 opened the pull request:
Removing all the old gitlab links to instead reference the appropriate places in GitHub.

joshsadam commented:
Can you also take a look at the following files:

- [ ] `ci/README.md`
- [ ] `doc/_site/developer/tools/pipelines/workflow-description/index.html`
- [ ] `doc/_site/developer/setup/index.html`
- [ ] `doc/_site/developer/getting-started/index.html`
- [ ] `doc/_site/developer/authorization/oauth2.html`
- [ ] `doc/_site/developer/rest/index.html`
- [ ] `doc/_site/feeds/feed.json`
- [ ] `doc/_site/administrator/web/building/index.html`
- [ ] `doc/_site/downloads/using-the-virtual-appliance.html`
- [ ] `pom.xml`

tom114 commented:
> Can you also take a look at the following files:
> 
> * [ ]  `ci/README.md`
> * [ ]  `doc/_site/developer/tools/pipelines/workflow-description/index.html`
> * [ ]  `doc/_site/developer/setup/index.html`
> * [ ]  `doc/_site/developer/getting-started/index.html`
> * [ ]  `doc/_site/developer/authorization/oauth2.html`
> * [ ]  `doc/_site/developer/rest/index.html`
> * [ ]  `doc/_site/feeds/feed.json`
> * [ ]  `doc/_site/administrator/web/building/index.html`
> * [ ]  `doc/_site/downloads/using-the-virtual-appliance.html`
> * [ ]  `pom.xml`

* Updated in `pom.xml`.  
* I added a bit in `ci/README.md` stating these are instructions for GitLab.  Still useful to keep around for now.  
* Everything in `doc/_site/` is generated by `mvn site` and isn't in the git repo (see https://github.com/phac-nml/irida/tree/development/doc).

7c6173f875
